tests/test_project.py

```diff
@@ -169,10 +170,22 @@ def test_get_project_settings(project_mock):
     assert project_settings[0]["name"] == "MAX_CONCURRENT_JOBS"
 
 
+def test_max_concurrent_jobs(project_mock, project_max_concurrent_jobs):
+    with project_max_concurrent_jobs(5):
+        max_concurrent_jobs = project_mock.max_concurrent_jobs
+    assert max_concurrent_jobs == 5
+
+
 @pytest.mark.live
 def test_get_project_settings_live(project_live):
     project_settings = project_live.get_project_settings()
     assert isinstance(project_settings, list)
     assert len(project_settings) == 3
     project_settings_dict = {item["name"]: item for item in project_settings}
     assert "MAX_CONCURRENT_JOBS" in project_settings_dict.keys()
+
+
+@pytest.mark.live
+def test_max_concurrent_jobs_live(project_live):
+    max_concurrent_jobs = project_live.max_concurrent_jobs
+    assert max_concurrent_jobs == 100
```

Comment:
Think typo, should be 10 right?

Reply:
Was also surprised, but if you look in the settings of the test account, it is set to 100.

Reply:
Okay interesting, good input for the discussion to increase it in general.